apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -268,6 +268,32 @@ const BookingPage = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const getBookingRedirectExtraParams = (booking: any) => {
+    if (!eventType.metadata?.passExtraParamsToRedirectUrl) {
+      return {};
+    }
+
+    const objectQueryParamKeys = ["customInputs", "responses", "metadata", "user", "attendees"];
+    const redirectQueryParamKeys = [
+      "title",
+      "description",
+      "startTime",
+      "endTime",
+      "location",
+      ...objectQueryParamKeys,
```

Comment:
Instead of differentiating manually b/w object and non-object params, we should use the typeof operator to identify the value. This would allow us to maintain a single list.

Also, I would like to keep the list very strict because whatever we expose has to be considered for backward compatibility as websites would rely on those params.

Stable Params - Good to be passed as is
- title
- description
- startTime
- endTime
- location


`user` and `attendees` might have different props depending on what we queried. So, we need to pick keys specifically from here. So, it might be better to remove these keys and wait for a requirement from users as to what they use it for.